src/guides/v2.3/frontend-dev-guide/themes/theme-inherit.md

```diff
@@ -169,3 +169,27 @@ To override the instructions from an ancestor theme layout file:
 To override module [layout instructions](https://glossary.magento.com/layout-instructions) (base layout):
 
 *  Create a layout file with the same name in the `<theme_dir>/<Vendor>_<Module>/layout/override/base` directory.
+
+**Example:**
+
+OrangeCo decided they should modify the product page layout completely to meet the new design, defined in `<Magento_Catalog_module_dir>/view/frontend/layout/catalog_product_view.xml`
```

Comment:
My only comment here is that OrangeCo exists. Suggest using a name that doesn't?

Reply:
@BarnyShergold  We need to use an imaginary company name that doesn't exist. Right?
If so, can we change the OrangeCo to something like _SampleCompany?_

Reply:
That would work @ajithkumar-maragathavel

Reply:
@BarnyShergold I have updated the company name. Please review it.